lite/valor_lite/segmentation/computation.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+from numpy.typing import NDArray
+
+
+def compute_intermediates(
```

Comment:
update this name to be more specific (e.g., `compute_intermediate_confusion_matrix`)